.travis.yml

```diff
@@ -1,45 +1,38 @@
 language: php
-sudo: required
-dist: trusty
 
 php:
   - 5.5
   - 5.6
   - 7.0
   - 7.1
-  - hhvm
+  - 7.2
 
 env:
```

Comment:
Isn't TYPO3 6.2 not working anymore ? If it's please add it to the matrix again, it should only be remove when not supported anymore.

Reply:
Done.